open-metadata-implementation/access-services/subject-area/subject-area-client/src/main/java/org/odpi/openmetadata/accessservices/subjectarea/client/nodes/SubjectAreaNodeClient.java

```diff
@@ -9,7 +9,7 @@
 
 /**
  * Annotating client class for working with Node objects.
- * These classes are scanned by {@link SubjectAreaNode}
+ * These classes are scanned by {@link DefaultSubjectAreaNodeClients}
```

Comment:
I am not clear why you call this a default. I would have thought that the scan would be standard and it picks up the content of the package using reflection. So we get classes loaded dynamically based on the scan. Are you envisaging a non-default scan - I am not sure what this would mean. Please let me know your thinking on the use of default - or remove it .

Reply:
Okay I remove this.